docs/collections/redis-mode/redis-data-format.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Redis Data Format
+sidebar_position: 50
+---
+
+Redis Mode collections require data to be stored in certain formats in order to be compatible with Redis.
+
+If you add items to the collection using the Redis API, then the format is enforced automatically. However, if you add items through Macrometa, then you must ensure that data follows the following formats according to type.
```

Comment:
Redis API is still part of macrometa. I would phrase the second sentence a little differently:

However, if you add items through other APIs, then you must ensure that data respects the following formats according to type.

Reply:
FYI, if you click the little page icon with the plus and minus, then you can add a suggestion directly.